en/04_Changelogs/4.13.0.md

```diff
@@ -75,4 +77,16 @@ If you are using this release as a step towards upgrading to Silverstipe CMS 5,
 
 ## Features and enhancements
 
+### GraphQL v4 additional fallback {#graphql-v4-additional-fallback}
```

Comment:
```suggestion
### GraphQL v4 additional fallback
```
I won't hold back merging for this, but as I've mentioned a few times before you don't need to add the anchor if it's identical to the default anchor.